contents/blog/0-to-hero.md

```diff
@@ -0,0 +1,93 @@
+1. then do a bigger launch - hacker news, to see if our target audience really cared
+
+There were a lot of things we did that made getting through the above easier:
+
+On the product side, we decided to focus on open source adoption and not revenue. I researched multi billion dollar open source companies and couldn't find a single example where they monetized in the first five years. I did this by cold messaging founders asking for advice and from what I could gather online. This meant we didn't have to worry about building a payment flow, we'd have slightly lower user expectations, or how to convert people to paying us from an open source product. However, this wouldn't be good advice universally. We were going to compete in a very crowded market of tools - with multiple competitors doing around $100M in revenue. That meant we could be confident we'd find a way to monetize. Had we been building into a new market, with less validation from competitors, then charging money sooner would have been important. Our challenge was _standing out_ enough, and being developer focused / open source we felt was enough.
+
+We set the product up to be quick to deploy - we created a Heroku one click deployment. Later on, we realized this was a mistake - we wound up eventually focusing on cloud adoption for most users. If I started again, I'd have a cloud product too even if it took a week or two longer. Finally, we picked a permissive license (MIT) because we wanted to see if enterprises would use it, and we felt they'd have rules around using non-permissive open source licenses, so we wanted to get rid of any risk they wouldn't use it.
+
+On the marketing side, we knew it was important to build trust. The foundation of trust is transparency, so we built a public handbook (and it has been a joy to maintain and expand as we've gotten a lot bigger). We ran a Hacker News premortem too.
+
+Marketing didn't need to be scalable to get the first 10 users. I asked my friends, only with the goal of building it _for_ someone with a real product (as we felt this would surface lots of edge cases etc, instead of us building it all locally at first).
+
+To find _your_ friends that might be relevant, start at the top and work you way down until you get 10 people to use your product:
+
+* whoever springs to mind first, as you likely know them the best
+* go through your whatsapp / texts
+* go through your linkedin, searching for people who are your first degree connections
+* go through your linkedin, looking for people who've ever worked at the same places you've worked
+
+You are unlikely to be self serve at this stage - if you are being very fast, you probably don't have the ability to create a new user without manually editing the database, so we would set people up over Slack / Whatsapp or physically with them. We did as much in person as we could.
+
+## Getting to 50 users - can people self serve (if doing product led growth) and does anyone care
```

Comment:
(Sorry I know this is still a very early draft - probably worth putting the 'product led growth' bit front and center at the top? Being able to self serve isn't relevant if I'm trying to get my first 50 compliance software customers for example)

Reply:
done